### PR TITLE
Make minor improvements to the "Payments" developer guides

### PR DESCRIPTION
**Pull request by benjaminwil**
**Description**

I've made some minor improvements to the "Payments" guide for Solidus developers. See the commit messages for more detailed change information.

**Checklist:**
- [x] I have followed [Pull Request guidelines](https://github.com/solidusio/solidus/blob/master/CONTRIBUTING.md#pull-request-guidelines)
- [x] I have added a detailed description into each commit message
- [x] I have updated Guides and README accordingly to this change (if needed)
- [x] ~~I have added tests to cover this change (if needed)~~
- [x] ~~I have attached screenshots to this PR for visual changes (if needed)~~

### CODE REVIEW

File: guides/source/developers/payments/payment-methods.html.md [hunk 1]
```diff
@@ -48,11 +48,25 @@ The base `Spree::PaymentMethod` class has just two preferences:
...
+
+### Behaviour when enabled
+
+When auto-capture is enabled, Solidus executes its payment capture flow as soon
+as the order is completed.
+
+### Behaviour when disabled
+
+When disabled, the payment remains uncaptured until a Solidus administrator (or
+your custom payment method code) manual executes the payment capture flow after
```
Comment (forkata) [inline guides/source/developers/payments/payment-methods.html.md:67]: `manual` should be `manually`

**Review by forkata — COMMENTED** [review]
Nice work @benjaminwil 🎊

**Review by adammathys — APPROVED** [review]

File: guides/source/developers/payments/payment-methods.html.md (hunk 1 above)
Comment (benjaminwil) [reply guides/source/developers/payments/payment-methods.html.md]: Thank you!

**Comment by benjaminwil** [comment]
I didn't make any code changes, but the spec suite is failing badly. I don't have permission to re-run the CircleCI pipelines.

**Review by spaghetticode — APPROVED** [review]
@benjaminwil thank you for these doc improvements ❤️

**Comment by jarednorman** [comment]
Oh, the failures are the google fonts shenanigans.

**Comment by benjaminwil** [comment]
I've rebased and specs are now passing.

**Review by jarednorman — APPROVED** [review]

**Pull request merged**